code/datums/spells/lichdom.dm

```diff
@@ -104,7 +104,7 @@
 
 			if(!marked_item)
 				M << "<span class='caution'>You must hold an item you wish to make your phylactery...</span>"
-
+				return
 			spawn(50)
```

Comment:
two procs of what
i dont understand anything anymore
git ruined my life
:sob: